src/enki_checks.py

```diff
@@ -1,8 +1,29 @@
 import re
+import itertools
 
 from enki_msg import Report
 from enki_regex import Regexes, Tags
 
+# standalone test to run on code blocks
+def sudo_check(
+    stripped_file: str,
+    report: Report,
+    file_path: str) -> None:
+    """Checks if sudo access is mentioned in documentation"""
+    code_block_dots = Regexes.CODE_BLOCK_DOTS.findall(stripped_file)
+    code_block_dashes = Regexes.CODE_BLOCK_DASHES.findall(stripped_file)
+    code_block_two_dashes = Regexes.CODE_BLOCK_TWO_DASHES.findall(stripped_file)
+
+    code_block = code_block_dots + code_block_dashes + code_block_two_dashes
+
+    # regex for root access  "# ", "~]# ", "*# ", " # ", "[root@", "[command]`# ", "#{nbsp}", "pass:quotes[#"
+    for item in ["$ sudo ", "$ *sudo ", "su -", "*su -"]:
+        for block in code_block:
+            if [b for b in block if b.startswith(item)]:
+                report.create_report(
```

Comment:
This is out of scope of this sudo PR, but have you considered augmenting the `create_report` method to also list line numbers of all the detected issues? I could contribute that functionality if you'd like.

Reply:
> This is out of scope of this sudo PR, but have you considered augmenting the `create_report` method to also list line numbers of all the detected issues? I could contribute that functionality if you'd like.

yep, I thought abt it too. the counter would be nice at this point

Reply:
Let's keep the counter in mind while we experiment with the tool tomorrow.